Setting up documentation for mflike

sgiardie opened the pull request:
I am setting up documentation for mflike. I have also added the placeholders for bandpass.py and foreground.py

itrharrison reviewed (changes requested):
The Bandpass documentation still needs work, I think. There are a number of functions which appear in the docs but have no comments associated with them.

Is it possible to remove the spellchecker underlining in `mflike_scheme.png`?

sgiardie commented on soliket/bandpass.py:
```diff
@@ -13,7 +13,9 @@
-
+    r"""
+    Computes the conversion factor :math:`\frac{\partial B_{\nu}}{\partial T}`
+    """
```
Done

sgiardie commented on soliket/mflike/mflike.py:
```diff
@@ -94,6 +120,12 @@ def loglike(self, cmbfg_dict):
+        trims the spectra and covariance according to the ell scales
+        set in the input file. It stores the ell vector, the deta vector
+        and the covariance in a GaussianData object.
+        """
```
Done

sgiardie commented:
> The Bandpass documentation still needs work, I think. There are a number of functions which appear in the docs but have no comments associated with them.

Indeed! I just added a line under the first function to try a math block and since it was working I left that there. If I remember well, @mgerbino was going to document both bandpass and foreground

> Is it possible to remove the spellchecker underlining in `mflike_scheme.png`?
I got that picture as a screenshot from a document that @mgerbino passed to me. Maybe she has the original?

sgiardie commented:
I have added documentation for `foreground.py` and `bandpass.py`. In the case of `bandpass.py` I added some comments due to the fact that some functions may change/be canceled with the upgrade of the mflike version.

itrharrison reviewed (approved):
LGTM!
